github/dependency_graph.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2023 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+	"time"
+)
+
+type DependencyGraphService service
+
+// Sbom represents software bill of materials, which descibes the
```

Comment:
"SBOM" should be all caps since it is an acronym... here, and throughout this PR.

Reply:
This change applies to comments as well as the code(such as function name) right?